src/stepFunctions/asl/aslServer.ts

```diff
@@ -283,11 +282,11 @@ function triggerValidation(textDocument: TextDocument): void {
 }
 
 // sets language service depending on document language
-function setLanguageService(textDocument: TextDocument): void {
-    if (textDocument.languageId == 'yasl') {
-        languageService = aslYamlLanguageService
+function getLanguageService(): LanguageService {
```

Comment:
I'm concerned about relying on a global var (`langId`). All logic depending on this function is at risk of error if different places in code set langId in close or overlapping proximity. Is there a way getLanguageService can act as a function of some input in all places this is needed?

Reply:
Ok, that makes sense. I'll update that